seed-job/template.yaml

```diff
@@ -35,7 +35,7 @@ objects:
                 containers: [
                   containerTemplate(
                     name: 'jnlp',
-                    image: 'registry.centos.org/dharmit/ccp-openshift-slave',
+                    image: '172.29.33.48:5000/nshaikh/ccp-jenkins-slave',
```

Comment:
why the IP address? Shouldn't it be a public image?

Reply:
I have made changes to container and queued it to build at r.c.o, however it was waiting in build queue, thus I locally built and pushed to my temp registry in devcloud. 

I will have it updated once I complete the work on this.